modules/account-lib/src/coin/baseCoin/baseTransactionBuilderFactory.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+
+export abstract class BaseTransactionBuilderFactory {

```

Comment:
since this is totally new - let's try and add some doc explanation what it's used for above it. for a nieve viewer the abstracted pattern might not be clear

Reply:
Done